gulpfile.coffee

```diff
@@ -0,0 +1,23 @@
+jshint = require 'gulp-jshint'
+stylish = require 'jshint-stylish'
+mocha = require 'gulp-mocha'
+coffee = require 'gulp-coffee'
+gulp   = require 'gulp'
+
+gulp.task "lint", ->
```

Comment:
Gulp works a ton better, as it is uses pipes to run tasks (which feels more like how Node stuff works), whereas Grunt used these crazy configuration objects.